doc/guides/pm/device_runtime.rst

```diff
@@ -154,7 +154,10 @@ must perform the necessary operations to suspend the device.
         ...
         /* make sure the device physically is suspended */
         /* enable device runtime power management */
-        pm_device_runtime_enable(dev);
+        ret = pm_device_runtime_enable(dev);
+        if (ret != 0) {
```

Comment:
in general `ret < 0` is used in the docs

Reply:
positive value is wrong though.

Reply:
well, API call claims to return a discrete set of negative values as error codes, so checking `< 0` covers all error scenarios (what this branch is testing for).